webservices/args.py

```diff
@@ -126,7 +126,9 @@ def make_seek_args(field=fields.Int, description=None):
 }
 
 query = {
-    'q': fields.Str(required=True, description='Text to search legal documents for')
+    'q': fields.Str(required=True, description='Text to search legal documents for'),
+    'from_hit': fields.Int(required=False, description='Get results starting from this index.'),
+    'hits_returned': fields.Int(required=False, description='Number of results to return (max 10).')
```

Comment:
Can we up the max, 10 seems pretty low for search. I would prefer a max of 200 with a default of 20, and include those in the docs.